ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -723,6 +723,60 @@ def test_tensorflow_asinh(
     )
 
 
+# acosh
+@handle_frontend_test(
+    fn_tree="tensorflow.math.acosh",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple([ivy.float32, ivy.float64]),
```

Comment:
Also, given that the functions have the proper dtype decorator, the dtypes for the tests should be specified as `available_dtypes=helpers.get_dtypes("valid")`.

Reply:
Thanks, changed that.